src/applications/ivc-champva/10-10D/pages/SignerContactInfoPage.jsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable react/sort-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import SchemaForm from '@department-of-veterans-affairs/platform-forms-system/SchemaForm';
+import {
+  titleUI,
+  titleSchema,
+  phoneSchema,
+  phoneUI,
+  emailSchema,
+  emailUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import FormNavButtons from '~/platform/forms-system/src/js/components/FormNavButtons';
+import { populateFirstApplicant } from '../helpers/utilities';
+import {
+  certifierPhoneValidation,
+  certifierEmailValidation,
+} from '../helpers/validations';
+
+export const signerContactInfoPage = {
+  uiSchema: {
+    ...titleUI(
+      'Your contact information',
+      'We use this information to contact you if we have more questions.',
+    ),
+    certifierPhone: phoneUI(),
+    certifierEmail: emailUI(),
+    'ui:validations': [certifierPhoneValidation, certifierEmailValidation],
+  },
+  schema: {
+    type: 'object',
+    required: ['certifierPhone', 'certifierEmail'],
+    properties: {
+      titleSchema,
+      certifierPhone: phoneSchema,
+      certifierEmail: emailSchema,
+    },
+  },
+};
+
+export function signerContactOnGoForward(props) {
+  const formData = props.data; // changes made here will reflect in global formData;
+  if (props?.data?.certifierRole === 'applicant') {
+    populateFirstApplicant(
+      formData,
+      formData.certifierName,
+      formData.certifierEmail,
+      formData.certifierPhone,
+      formData.certifierAddress,
+    );
+  } else if (props?.data?.certifierRole === 'sponsor') {
+    // Populate some sponsor fields with certifier info:
+    formData.sponsorIsDeceased = false;
+    formData.veteransFullName = formData.certifierName;
+    formData.sponsorAddress = formData.certifierAddress;
+    formData.sponsorPhone = formData.certifierPhone;
+  }
+
+  if (formData?.applicants?.[0]) {
+    // This is so we have an accurate `certifierRole` prop inside the
+    // list loop where that context is normally unavailable:
+    formData.applicants[0]['view:certifierRole'] = formData?.certifierRole;
+  }
+}
```

Comment:
Custom `onGoForward` function - when the user clicks `continue` on this custom page, any data we want to duplicate is sent to the appropriate fields downstream.